Extension/src/LanguageServer/settings.ts

```diff
@@ -287,8 +287,8 @@ export class CppSettings extends Settings {
     }
 
     public get enhancedColorization(): boolean {
-        return super.Section.get<string>("enhancedColorization") === "Enabled"
-            && super.Section.get<string>("intelliSenseEngine") === "Default"
+        return super.Section.get<string>("enhancedColorization") === "enabled"
+            && super.Section.get<string>("intelliSenseEngine") === "default"
             && vscode.workspace.getConfiguration("workbench").get<string>("colorTheme") !== "Default High Contrast";
     }
 
```

Comment:
note that there is also a reference in a string to 'Default' on line 756 of settings.ts that needs to be updated.

Reply:
do these comparisons need to be forced to lowercase as well?

Reply:
I see all comparison need to be converted, gotcha